### PR TITLE
📚 Docs Refresh: Postgres Source

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
- Cleaned up a lot of unnecessary information
- Reorganized structure
- Retitled sections for clarity
- Quality of life improvements

### CODE REVIEW

File: docs/integrations/sources/postgres.md [hunk 1]
```diff
@@ -81,18 +11,24 @@ If you can't find the data type you are looking for or have any problems feel fr
 | Logical Replication \(WAL\) | Yes |  |
 | SSL Support | Yes |  |
 | SSH Tunnel Connection | Coming soon |  |
-| Namespaces | Yes | Enabled by default |
+| Namespaces | Yes | Enabled by default. |
+| Custom Types | Yes |  |
+| Arrays | Yes | Byte-arrays are not supported. |
```
Comment (sherifnada) [inline docs/integrations/sources/postgres.md:16]:
```suggestion
| Arrays | Yes | Byte-arrays are not supported yet. |
```

File: docs/integrations/sources/postgres.md [hunk 2]
```diff
@@ -81,18 +11,24 @@ If you can't find the data type you are looking for or have any problems feel fr
...
+| Generating an RSA Private Key | No | Coming Soon. |
 
-## Getting started
+The Postgres source does not alter the schema present in your database. Depending on the destination connected to this source, however, the schema may be altered. See the destination's documentation for more details.
+
+## Getting Started (Airbyte Cloud)
+On Airbyte Cloud, only TLS connections to your Postgres instance are supported in source configuration. Other than that, you can proceed with the open-source instructions below.
```
Comment (sherifnada) [inline docs/integrations/sources/postgres.md:22]:
```suggestion
On Airbyte Cloud, only TLS connections to your Postgres instance are supported. Other than that, you can proceed with the open-source instructions below.
```

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/postgres.md [hunk 3]
```diff
@@ -81,18 +11,24 @@ If you can't find the data type you are looking for or have any problems feel fr
 | Logical Replication \(WAL\) | Yes |  |
 | SSL Support | Yes |  |
 | SSH Tunnel Connection | Coming soon |  |
```
Comment (marcosmarxm) [inline docs/integrations/sources/postgres.md:13]: This is not supported now?

**Review by marcosmarxm — APPROVED** [review]
LGMT small comment about SSH feature.

File: docs/integrations/sources/postgres.md (hunk 3 above)
Comment (avaidyanatha) [reply docs/integrations/sources/postgres.md]: It is! It just wasn't updated along with the addition of the SSH tunnel docs, updating now, thanks :)

**Pull request merged**